js-objects/objects.js

```diff
@@ -0,0 +1,115 @@
+// Task 1:
+let automationQA = {};
+
+automationQA.programmerName = "Ivan";
+automationQA.programmingLanguages = "Javascript";
+automationQA.age = 30;
+automationQA.languages = ["English", "Ukrainian", "Bulgarian"];
+automationQA.preferences = {basicPreference: "table"};
+
+console.log(automationQA);
+
+// Task 2:
+let room = {
+    name : "Living room",
+    size : 15,
+    window : "true",
+}
+console.log(room.name, "," , "window - ", room.window);
+
+// Task 3:
+let movie = {
+    title: "American Pie",
+    releaseYear: 2013,
+    genre: "Comedy",
+
+}
+
+movie.genre = "Fantastic";
+movie.title = "Home Alone";
+
+movie.actors = {mainRole: "Test Test", additionalRole: "One additional"};
+movie.director = "Some director";
+
+delete movie.title;
+console.log(movie);
+
+// Task 4:
+const englishBooks = [
+    {
+        title: "To Kill a Mockingbird",
+        author: "Harper Lee",
+        year: 1960
+    },
+    {
+        title: "1984",
+        author: "George Orwell",
+        year: 1949
+    },
+    {
+        title: "The Great Gatsby",
+        author: "F. Scott Fitzgerald",
+        year: 1925
+    }
+];
+
+for (const book of englishBooks) {
+
+   for (const prop in book) {
+    let val = book[prop];
+   if (val === "The Great Gatsby") {
+        continue;  
+    }
+   if (val > "1940" && typeof val === "number") {
+    console.log("> 1940", val);
+   }
+    if (val === "George Orwell")
+      book[prop] = "New author";
+}
+}
```

Comment:
Add formatting here. 
```suggestion
    } 
}
```